tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js

```diff
@@ -0,0 +1,164 @@
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+
+     // Click to 3rd checkbox, check disabled, click once again, check disabled again
+     loginTermsPage.checkboxes[2].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[2].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+
+     // Click to 4th checkbox, check disabled, click once again, check disabled again
+     loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+  })
+
+  //Step 3. Check  any 2 checkboxes(for example - 1, 2, 4) and try to click -> 'Continue' is disabled
+  it('Check  any 2 checkboxes', () => {
+     //Click to 1st and 2nd checkboxes, check disabled, click once again, check disabled again
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[1].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[1].click()
+
+     //Click to 1st and 3nd checkboxes
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[2].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[2].click()
+
+     //Click to 1st and 4th checkboxes
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[0].click()
+     loginTermsPage.checkboxes[3].click()
+
+     //Click to 2st and 3th checkboxes
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[2].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[2].click()
+
+     //Click to 2st and 4th checkboxes
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[3].click()
+
+     //Click to 3rd and 4th checkboxes
+      loginTermsPage.checkboxes[2].click()
+      loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[2].click()
+     loginTermsPage.checkboxes[3].click()
+  })
+
+  //3. Check  any 3 checkboxes(for example - 1, 2, 4) and try to click -> 'Continue' is disabled
+  it('', () => {
```

Comment:
What are we testing here?

Reply:
We are checking that the 'Continue' button is disabled and it's not possible to proceed with only 3 checked checkboxes. Do you think that these tests are redundant?

Reply:
There is nothing in test description, just empty quotes